include/inset_state.h

```diff
@@ -18,17 +19,31 @@ struct max_area_error_info {
   std::string geo_div;
 };
 
-class InsetState {
+class InsetState
+{
 private:
   std::unordered_map<std::string, double> area_errors_;
-  Bbox bbox_;  // Bounding box
+  Bbox bbox_; // Bounding box
   fftw_plan bwd_plan_for_rho_;
   std::unordered_map<std::string, Color> colors_;
 
+  // Unblurred density mean, min, max
+  double dens_min_;
+  double dens_mean_;
+  double dens_max_;
+
+  // Scaling factor to convert albers unit to 512*512 unit
+  double latt_const_;
+
   // Cumulative cartogram projection
   boost::multi_array<XYPoint, 2> cum_proj_;
   fftw_plan fwd_plan_for_rho_;
-  std::vector<GeoDiv> geo_divs_;  // Geographic divisions in this inset
+
+  // Geographic divisions in this inset
+  std::vector<GeoDiv> geo_divs_;
+
+  // Copy of original data
+  std::vector<GeoDiv> geo_divs_original_;
```

Comment:
```geo_divs_original_``` is needed for the equal-area map graticule heatmap since we can only work on it once the integrations are done. I created that getter function, but we never ended up using it. Later, we moved map drawing functions to ```inset_state``` member and allowed us to access ```inset_state`` private variables directly, making the getter function obsolete.